### PR TITLE
Rs 2022.11.16.punctuation

### PR DESCRIPTION
**Pull request by rschlaikjer**
Chipping away

### CODE REVIEW

File: script/Arcueid/Day 2/02_00_ARC02_1.txt [hunk 1]
```diff
@@ -357,7 +357,7 @@ However―――
 [sha:385490faa541e70e5790e052be63927aa013436c]{
 -- Page 24, Offset 2664.
 -- 「……しまった。昨日のうちに言っておくべきだった」
-"...Oh no. I should have told you yesterday."
+"I'm sorry. I should have told you yesterday."
```
Comment (Petrikow) [inline script/Arcueid/Day 2/02_00_ARC02_1.txt:360]: Probably works  more like "my bad" or something.

File: script/Arcueid/Day 2/02_00_ARC02_6.txt [hunk 2]
```diff
@@ -611,17 +613,17 @@ I manage to fall onto my bed.
...
 -- Page 372, Offset 3612.
 -- ……体も思うように動かない。
-...I can't even move my body like I want to.
+My body won't even do as I tell it...
 }
 [sha:f61ece60f8b5a16771b841f92b3350faa527967b]{
 -- Page 372, Offset 3613.
 -- 今はこのまま死体のように。
-For now, I'm just going to stay like this, like a corpse.
+I'll just stay like this, spread out like a corpse.
```
Comment (Petrikow) [inline script/Arcueid/Day 2/02_00_ARC02_6.txt:626]: There's another like below.
Maybe worth considering.

**Review by Petrikow — COMMENTED** [review]
cxv

File: script/Arcueid/Day 2/02_00_ARC02_1.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 2/02_00_ARC02_1.txt:360]: Yeah, that's the vibe I read too, but starting the topic with 'my bad' feels off to me. More something you'd say once it's already established what's going on. "Ah, crap" was my first thought but no swearing at the dinner table. "Ah, shoot"? Or too charlie brown?

**Pull request merged**